cloud

```diff
@@ -1243,6 +1243,20 @@ function pubsub_withdraw() {
     pubsub_topic_withdraw "$project" "$pick_notifications_trigger_topic"
 }
 
+function storage_deploy(){
+    
+    gsutil mb -p "$project" -c STANDARD -l "us-central1" "gs://$storage_bucket"
+
+    gsutil iam ch allUsers:objectViewer "gs://$storage_bucket/"
+
```

Comment:
Please remove unnecessary empty lines around the inside of the function body (this one, and the one at the beginning). In the other function as well.

Reply:
Okay